src/menus

```diff
@@ -7,6 +7,7 @@ p({"class":"foam.nanos.menu.Menu","id":"export","label":"Export Drivers","handle
 p({"class":"foam.nanos.menu.Menu","id":"groups","label":"Groups","handler":{"class":"foam.nanos.menu.DAOMenu","daoKey":"groupDAO"},"order":1000,"parent":"admin"})
 p({"class":"foam.nanos.menu.Menu","id":"lang","label":"Languages","handler":{"class":"foam.nanos.menu.DAOMenu","daoKey":"languageDAO"},"order":1000,"parent":"admin"})
 p({"class":"foam.nanos.menu.Menu","id":"log","label":"View Logs","order":1000,"parent":"admin"})
+p({"class":"foam.nanos.menu.Menu","id":"alterna-eft","label":"Alterna-EFT","handler":{"class":"foam.nanos.menu.ViewMenu", "view":{"class":"net.nanopay.cico.ui.AlternaEFTDownload"}},"order":1000,"parent":"admin"})
```

Comment:
Is AlternaEFTDownload checked in?

Reply:
This references a class that's not in foam2

Reply:
yea it works fine

Reply:
it was a request from laurence and varun, I could make the file be in foam2 but i didnt think it was necessary